src/naett_linux.c

```diff
@@ -193,7 +193,11 @@ void naettPlatformMakeRequest(InternalResponse* res) {
     setupMethod(c, req->options.method);
 
     struct curl_slist* headerList = NULL;
-    headerList = curl_slist_append(headerList, "User-Agent: Naett/1.0");
+    char uaBuf[512];
+    if (req->options.userAgent) {
```

Comment:
The default `NAETT_UA` will never be set here.

Reply:
Oops, silly of me. Removing the `if`.